charts/prometheus-rabbitmq-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 1.0.0
+version: 1.0.1
```

new feature should raise minor version
```suggestion
version: 1.1.0
```

---

Thanks, I've updated it now.